docs/block-development/extensible-blocks/cart-and-checkout-blocks/checkout-payment-methods/payment-method-integration.md

```diff
@@ -148,15 +148,114 @@ registerPaymentMethod( options );
-| `ariaLabel` | string | The label read by screen-readers when the payment method is selected. |
-| `placeOrderButtonLabel` | string | Optional label to change the default "Place Order" button text when this payment method is selected. |
-| `supports` | object | Contains information about supported features: |
-| `supports.showSavedCards` | boolean | Determines if saved cards for this payment method are shown to the customer. |
-| `supports.showSaveOption` | boolean | Controls whether to show the checkbox for saving the payment method for future use. |
+| Property                  | Type              | Description                                                                                                                                      |
+|---------------------------|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
+| `savedTokenComponent`     | ReactNode         | A React node that contains logic for handling saved payment methods. Rendered when a customer's saved token for this payment method is selected. |
+| `label`                   | ReactNode         | A React node used to output the label for the payment method option. Can be text or images.                                                      |
+| `ariaLabel`               | string            | The label read by screen-readers when the payment method is selected.                                                                            |
+| `placeOrderButtonLabel`   | string            | Optional label to change the default "Place Order" button text when this payment method is selected. Mutually exclusive with `placeOrderButton`. |
+| `placeOrderButton`        | React Component   | Optional React component to replace the default "Place Order" button when this payment method is selected. Mutually exclusive with `placeOrderButtonLabel`. The component receives `PaymentMethodInterface` props. |
+| `supports`                | object            | Contains information about supported features:                                                                                                   |
+| `supports.showSavedCards` | boolean           | Determines if saved cards for this payment method are shown to the customer.                                                                     |
+| `supports.showSaveOption` | boolean           | Controls whether to show the checkbox for saving the payment method for future use.                                                              |
+
+### Using Custom Place Order Button
+
+The `placeOrderButton` property allows you to replace the default "Place Order" button with a custom component. This is useful for payment methods that require custom button styling (e.g., Google Pay, Apple Pay) or need to show a payment UI before submitting the order.
```

Comment:
Should we include some writing "discouraging" this unless necessary? For example add: "If this doesn't apply to you, omitting this prop and using the default button is recommended"